extras/kubernetes/heketi-deployment.json

```diff
@@ -1,119 +1,142 @@
 {
-  "kind": "Service",
+  "kind": "List",
   "apiVersion": "v1",
-  "metadata": {
-    "name": "heketi",
-    "labels": {
-      "glusterfs": "heketi-service",
-      "deploy-heketi": "support"
-    },
-    "annotations": {
-      "description": "Exposes Heketi Service"
-    }
-  },
-  "spec": {
-    "selector": {
-        "name": "heketi"
+  "items": [
+    {
+      "kind": "Secret",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "heketi-db-backup",
+        "labels": {
```

Comment:
I'd say don't include the labels yet if we don't need them, they're unnecessarily very specific and it would make the secret JSON simpler if they were omitted.